starknet_py/tests/e2e/docs/guide/test_serializing.py

```diff
@@ -0,0 +1,45 @@
+# pylint: disable=import-outside-toplevel
+import json
+
+from starknet_py.tests.e2e.fixtures.misc import read_contract
+
+
+def test_abi_parsing():
+    raw_abi_string = read_contract("erc20_abi.json")
```

Comment:
Let's include the ABI as multiline string in the docs so it's obvious what abi is being used.

You can use only parts of erc20 abi that are necessary for the example.